test/functional/mempool_accept_v3.py

```diff
@@ -312,7 +312,8 @@ def test_mempool_sibling(self):
             version=3
         )
         expected_error_mempool_sibling = f"v3-rule-violation, tx {tx_mempool_parent['txid']} (wtxid={tx_mempool_parent['wtxid']}) would exceed descendant count limit"
-        assert_raises_rpc_error(-26, expected_error_mempool_sibling, node.sendrawtransaction, tx_has_mempool_sibling["hex"])
+        expected_error_mempool_sibling_no_eviction = f"insufficient fee (including sibling eviction), rejecting replacement"
```

Comment:
log/description of this test should get updated or maybe merged into test_v3_sibling_eviction

Reply:
I've repurposed this test to check that sibling eviction is not allowed through multi-testmempoolaccept or submitpackage